ports/yolkheroes/LICENSE.FRT.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
This file needs to go inside the next yolkheroes folder, it's why the pr is failing on build